### PR TITLE
Register plugins

### PR DESCRIPTION
**Pull request by jay-meister**
Register other pugins into this one. fix tests.

### CODE REVIEW

File: tests/orgs/getDetails/endpoint.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// 'use strict';
```
Comment (SimonLab) [inline tests/orgs/getDetails/endpoint.test.js:1]: do you want to delete this file, or are you going to reuse it?

File: lib/formatters/orgsGetDetails.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = function (rows) {
```
Comment (SimonLab) [inline lib/formatters/orgsGetDetails.js:3]: This formater might need a bit more code, I guess it will come in the next PR.

**Comment by jay-meister** [comment]
yeah you are right on both counts. Sorry I didn't know of a way to filter out a few files quickly.

**Review by SimonLab — COMMENTED** [review]
Looks good, do you want to update the version of the package, thanks

**Review by SimonLab — APPROVED** [review]
Looks good

**Pull request merged**